internal/models/cluster/nodepool/nodepools_spec_tkg_aws.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package nodepool
+
+import (
+	"github.com/go-openapi/swag"
+)
+
+// VmwareTanzuManageV1alpha1ClusterNodepoolTKGAWSNodepool TKGAWSNodepool is the nodepool spec for TKG aws cluster.
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.cluster.nodepool.TKGAWSNodepool
+type VmwareTanzuManageV1alpha1ClusterNodepoolTKGAWSNodepool struct {
+
+	// Availability zone for the nodepool. Should be one of the availability zones chosen for the cluster.
+	// Use this field only if you are creating a nodepool for cluster in TMC hosted AWS solution. To create a nodepool for TKG
+	// workload cluster please use TKGAWSNodePlacement
+	AvailabilityZone string `json:"availabilityZone,omitempty"`
+
+	// Nodepool instance type.
+	// The potential values could be found using cluster:options api.
+	InstanceType string `json:"instanceType,omitempty"`
+
+	// List of AZs to place the AWS nodes on. Please use this field to provision a nodepool for workload cluster on an attached TKG AWS management cluster.
+	// Please specify 1 AZ for a dev cluster and up to 3 AZs for production cluster.
+	NodePlacement []*VmwareTanzuManageV1alpha1ClusterNodepoolTKGAWSNodePlacement `json:"nodePlacement"`
+
+	// The nodes private CIDR block if we are creating a new VPC.
+	// This is no longer used - Please dont use this deprecated field.
+	PrivateCidr string `json:"privateCidr,omitempty"`
```

Comment:
nit: `PrivateCIDR`

Reply:
Also, since it is deprecated should we remove it?

Reply:
@gsadhani the spec attributes int he model files are autogenerated using the go swagger tool, these variables are not manually named here. Not used these specs in our resources and examples, but can be removed from here too.